requirements/build.txt

```diff
@@ -1,2 +1,2 @@
 cython
-numpy
+numpy<1.22
```

Comment:
We may limit the version of numba to 0.53.0 following [mmdet3d](https://github.com/open-mmlab/mmdetection3d/blob/master/requirements/runtime.txt#L3)